tests/acceptance/lang-path-test.js

```diff
@@ -0,0 +1,28 @@
+import Service from '@ember/service';
+import {find, findAll, fillIn, triggerEvent, visit, currentURL} from '@ember/test-helpers';
+import {setupApplicationTest} from 'ember-qunit';
+import {module, test} from 'qunit';
+import mockFastbootService from '../helpers/mock-fastboot-service';
+
+const wikiUrlsServiceStub = Service.extend({
+	langPath: '/pl'
+});
+
+module('Acceptance | lang path', (hooks) => {
+	setupApplicationTest(hooks);
+
+	hooks.beforeEach(function () {
+		mockFastbootService(this.owner);
+		this.owner.register('service:wikiUrls', wikiUrlsServiceStub);
+	});
+
+	test('visiting /pl/wiki/test', async (assert) => {
+		const result = await visit('/pl/wiki/test');
+		assert.ok(result);
```

Comment:
how about asserting if currentURL is the one that is expected?

assert.ok - can assert on promises? neat!

Reply:
Good idea, added in https://github.com/Wikia/mobile-wiki/pull/618/commits/0193b4105653b8ec5ee4fc135651e246621ccc66